docs/en/sql-reference/functions/string-functions.md

```diff
@@ -278,6 +306,34 @@ Does not detect the language, e.g. for Turkish the result might not be exactly c
 
 If the length of the UTF-8 byte sequence is different for upper and lower case of a code point, the result may be incorrect for this code point.
 
+**Syntax**
+
+``` sql
+upperUTF8(input)
+```
+
+**Parameters**
+
+- `input`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+
+**Returned value**
+
+- A [String](/docs/en/sql-reference/data-types/string.md) data type value.
+
+**Example**
+
+Query:
+
+``` sql
+SELECT upperUTF8('value') as Upperutf8;
```

Comment:
An example involving non-ASCII UTF-8 would be nice. Maybe

```sql
SELECT upperUTF8('München');

Result MÜNCHEN
```

Reply:
Updated. Thanks for your recommendation.